yaml/com/ibm/Panel.interface.yaml

```diff
@@ -0,0 +1,85 @@
+description: >
+    Implements control panel activities.
+methods:
+    - name: Display
+      description: >
+          Commands the op-panel to display the given lines.
+      parameters:
+          - name: line1
+            type: string
+            description: >
+                First line that needs to be displayed in op-panel LCD.
+          - name: line2
+            type: string
+            description: >
+                Second line that needs to be displayed in op-panel LCD.
+
+    - name: TriggerPanelLampTest
+      description: >
+          Commands the op-panel to trigger panel lamp test.
+      parameters:
+          - name: status
+            type: boolean
+            description: >
+                Panel lamp test request.
+
+    - name: toggleFunctionState
+      description: >
+          Toggles the state of given panel functions.
+      parameters:
+          - name: bitmap
+            type: array[byte]
+            description: >
+                Bit number in bitmap will have one to one mapping with function
+                number. Bit value 1 to enable and 0 to disable the function.
+
+    - name: ProcessButton
+      description: >
+          Method which simulates panel button request.
+      parameters:
+          - name: event_id
+            type: int32
+            description: >
+                Event ID = 0 simulates UP button. Event ID = 1 simulates DOWN
+                button. Event ID = 2 simulates ENTER button.
+
+    - name: ExecuteFunction
+      description: >
+          Trigger panel function directly.
```

Comment:
Reword? API to trigger panel function.

Directly does not make any sense here.